bin/send_email.py

```diff
@@ -1,41 +1,18 @@
 #!/usr/bin/env python3
 
 import sys
-import synapseclient
-
 from typing import List, NamedTuple
 
+import synapseclient
+
+import helpers
 
 class SubmissionAnnotations(NamedTuple):
     status: str
     score: List[int]
     reason: str
 
 
-def get_participant_id(syn: synapseclient.Synapse, submission_id: str) -> List[str]:
```

Comment:
I think this change is going to break the email [script](https://github.com/Sage-Bionetworks-Workflows/nf-synapse-challenge/blob/main/bin/dynamic_challenge_send_email.py) for the dynamic challenge. Be sure to merge main back into this branch and address this before merging

Reply:
If this is the case, it's a pretty good example of why we shouldn't have the scripts being called by Nextflow processes be dependent on each other (my bad 😅)

Reply:
Hi @BWMac thanks for the heads up - I've updated this branch with the correct import and rebased this branch onto `main`. [Here](https://tower-dev.sagebionetworks.org/orgs/Sage-Bionetworks/workspaces/example-dev-project/watch/2e631ntM4yMuZg) is a `tower-dev` run for the `DATA_TO_MODEL` workflow pointed at the latest version of this PR.